### PR TITLE
Update LocalCacheFileSystem

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/client/hdfs/src/main/java/alluxio/hadoop/LocalCacheFileSystem.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
  */
 public class LocalCacheFileSystem extends org.apache.hadoop.fs.FileSystem {
   private static final Logger LOG = LoggerFactory.getLogger(LocalCacheFileSystem.class);
-  private static final Set SUPPORTED_FS = new HashSet<String>() {
+  private static final Set<String> SUPPORTED_FS = new HashSet<String>() {
```
Comment (calvinjia) [inline core/client/hdfs/src/main/java/alluxio/hadoop/LocalCacheFileSystem.java:53]: `HashSet<>`?

Comment (apc999) [reply core/client/hdfs/src/main/java/alluxio/hadoop/LocalCacheFileSystem.java:53]: Required by Java when having annonymouse inner class

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.2.2 please

**Pull request merged**